mis_builder_budget/__manifest__.py

```diff
@@ -5,7 +5,7 @@
     "name": "MIS Builder Budget",
     "summary": """
         Create budgets for MIS reports""",
-    "version": "14.0.4.0.0",
+    "version": "15.0.4.0.0",
```

Comment:
```suggestion
    "version": "15.0.1.0.0",
```

Reply:
this should be like that. @sbidoul pointed here: https://github.com/OCA/mis-builder/pull/406#discussion_r794293959

Reply:
Ok, strange. I don't understand why

Reply:
@leemannd because I try to maintain feature parity across Odoo versions, and the major and minor version number communicates that. TBH, resetting to 1.0 when we change Odoo version is what looks *really* strange to me, but :shrug: there are fights I have abandonned.

Reply:
@sbidoul Thanks for the answer. I was surprised has it is not commonly done in other modules in OCA.
On this topic I understand both point of views. But IMO your method is best suited if there is a maintainer that has the time to care (and has the time) about _maintaining_ the code as a library. Which seems to be the case

Reply:
@leemannd yes, exactly that. And I sure wish there were more real maintainers in OCA.